doc/user_doc/vic_installation/ts_ui_not_appearing.md

```diff
@@ -32,4 +31,10 @@ Restart the vSphere Web Client service.
    - vCenter Server 6.0:<pre>service vsphere-client start</pre>
    - vCenter Server 6.5: <pre>service-control --start vsphere-client</pre>
 
+### Restart the HTML5 or Flex Client for vSphere 6.5 ###
+
+1. Log into the Flex-based vSphere Web Client.
+2. Got to **Home** > **System Configuration** > **Services** > **VMware vSphere Web Client**.
```

Comment:
Service name for Flex Client is **VMware vSphere Web Client** and **VMware vSphere Client** for H5 Client